ODBC.pm

```diff
@@ -2577,20 +2577,6 @@ schema required and the versions of software you are using.
 If you are unsure whether you have found a bug report it anyway or
 post it to the dbi-users mailing list.
 
-=item pod comments and corrections
-
-If you find inaccuracies in the DBD::ODBC pod or have a comment which
-you think should be added then go to L<http://annocpan.org> and submit
-them there. I get an email for every comment added and will review
-each one and apply any changes to the documentation.
```

Comment:
Perhaps this should be edited to provide different instructions for the described feedback (github issues, maybe?), rather than simply deleted, leaving no path for that feedback?

Reply:
I thought about re-writing stating that feedback for POD can be provided via Github. But the github repo is already linked from the module metadata and various other places.

annocpan came from a time where there was no github and collaboration was much more difficult, svn.perl.org  n- where this module was hosted previously - was 'read only' for everybody except the author and you had to have a 'commit bit' for pushing and pull requests did not exists, they were diff files. 

Of course it might still be desirable to have explicit text soliciting feedback for the POD; and that could definitely added, or the original paragraph can be re-phrased, or whatever. The PR was just for 1. pointing out that the links were invalid, 2. adding a one-click way to remove those invalid links.